src/scielo/bin/xml/prodtools/xc.py

```diff
@@ -157,25 +157,36 @@ def convert_package(self, package_path):
 
         output_path = mkdtemp()
 
-        pkg = self._create_package_instance(source=xml_path, output=output_path)
-        pkg_name = pkg.package_folder.name
-
         try:
-            result = self.proc.convert_package(pkg)
-            scilista_items, xc_status, mail_info = result
-        except Exception as e:
+            package = self._create_package_instance(source=xml_path, output=output_path)
+            scilista_items, xc_status, mail_info = self.proc.convert_package(package)
+        except ScieloPackageError:
+            self.inform_failure(
+                package_path,
+                "Could not create package from source path '%s'." % package_path,
+            )
```

Comment:
A exceção tem que ser logada para facilitar a correção.

Reply:
Pensei que deveria manter o mesmo esquema utilizado antes do PR, @robertatakenaka. Alterarei.

Reply:
entendi... há um issue aberto só para tratar das exceções deste módulo: #3230. Então, poderia deixar para quem for resolver o ticket de dar o tratamento adequado.

Reply:
@robertatakenaka eu vou continuar tratando essas exceções, é mais seguro para o programa. Caso algum trecho não seja coberto poderia ocorrer uma quebra de execução.

Vou melhorar o log e manter o e-mail também.

Reply:
@robertatakenaka a alteração foi feita em 979ce3f7dce8349999682024c06a1b0a4bccc6a0